.editorconfig

```diff
@@ -0,0 +1,22 @@
+root = true
+
+[*]
+indent_style = tab
+indent_size = tab
+tab_width = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+# Ensure text editors don't turn leading spaces into tabs,
+# e.g. in multi-line bullet list items
+[*.md]
+indent_style = space
+indent_size = 2
+
+# Tabs may not be valid YAML
+# @see https://yaml.org/spec/1.2/spec.html#id2777534
+[*.{yml,yaml}]
+indent_style = space
+indent_size = 2
```

Comment:
This file is changing how GitHub displays the tabs. It went from 4 spaces width to 2. Bit weird given the config here, which says 4...

Reply:
After some investigation,  it seems to only be affecting diffs, not the actual code display.  

Compare [view file of src/Application/Mapping.php](https://github.com/ProfessionalWiki/WikibaseRDF/blob/cf20ca4617ef8ec6eef99de6e325374cec16d58b/src/Application/Mapping.php) with a [that commit's diff](https://github.com/ProfessionalWiki/WikibaseRDF/commit/cf20ca4617ef8ec6eef99de6e325374cec16d58b#diff-0c9480a803a414b407c4430c457b8a6cea898c81a035c384b07f212b21663888).

This is a github bug, it looks like.  [This so comment](https://stackoverflow.com/a/33899831) is relevant.